src/handler.rs

```diff
@@ -23,9 +24,15 @@ use virtio_bindings::bindings::virtio_ring::VIRTIO_RING_F_EVENT_IDX;
 use vm_memory::bitmap::Bitmap;
 use vm_memory::mmap::NewBitmap;
 use vm_memory::{
-    FileOffset, GuestAddress, GuestAddressSpace, GuestMemoryMmap, GuestRegionMmap, MmapRegion,
+    Address, FileOffset, GuestAddress, GuestAddressSpace, GuestMemoryMmap, GuestRegionMmap,
+    MmapRegion,
 };
 
+use libc::c_void;
+use std::mem;
+use std::os::unix::io::RawFd;
```

Comment:
better to move line 32-33 to near line 11. So we could group dependency as
- std
- external crate
- self crate